lib/ngsiHandlers.js

```diff
@@ -34,6 +34,34 @@ var iotAgentLib = require('iotagent-node-lib'),
         op: 'IOTAgent.NGSIHandlers'
     };
 
+// Ezequiel:
+function extractOperations(device, attribute) {
+    var operations;
+
+    if (device.internalAttributes && device.internalAttributes.lwm2mResourceMapping &&
+        device.internalAttributes.lwm2mResourceMapping[attribute]) {
+        operations = device.internalAttributes.lwm2mResourceMapping[attribute].operations;
+    }
+    if (!operations && config.ngsi.types[device.type] &&
+        config.ngsi.types[device.type].lwm2mResourceMapping &&
+        config.ngsi.types[device.type].lwm2mResourceMapping[attribute])
+        operations = config.ngsi.types[device.type].lwm2mResourceMapping[attribute].operations;
+    if (!operations && omaInverseRegistry[attribute])
+        operations = omaInverseRegistry[attribute].operations;
+    if (!operations && attribute.indexOf('#') >= 0) {
+        var attributeName = attribute.substr(0, attribute.indexOf('#')),
+            resourceNumber = attribute.substr(attribute.indexOf('#') + 1),
+            temporalMapping = omaInverseRegistry[attributeName];
+
+        if (temporalMapping) {
+            operations = temporalMapping.operations;
+        }
+    }
+    //TODO: ¿Por defecto se permite todo?
```

Comment:
We can assume that for the moment. Please, if you think a TODO tag is worth keeping in some part of the code, translate it to english.